scripts/fill_db.py

```diff
@@ -1,41 +1,42 @@
 # general utils
 import json
+# for the randomness
 import os
-import sys
+import random
 import string
+import sys
 
-# for the randomness
-import random
 import names
-import randomname
 import numpy as np
-
+# add the models package to the path
+import path
+import randomname
 # flask imports
 from flask import Flask
 from flask_mongoengine import MongoEngine
 
-# add the models package to the path
-import path
 directory = path.Path(__file__).abspath()
 sys.path.append(directory.parent.parent)
 from models import *
 
+if len(sys.argv) != 3:
+    print('Usage: python3 fill_db.py [admin-username] [admin-password]')
+    sys.exit()
+
```

Comment:
The only argument taken to this file is the optional configuration file. by default, it should use the data_gen_config.json, but if a user specifies a config file it should use that

When this file is called the API user should have already been made in the DB and in the environment variables, so the admin username and password are not needed.

Reply:
The reasoning for adding admin username and admin password to this call is such that we can lock the admin user after the initial setup is complete on production builds (we'll implement that functionality later, but this is to prepare for it)

There should be no script that is able to tamper with the database (such as adding data) without admin credentials on a debug build. Supplying API user through environment variables will have be removed in the future, thus any initial database operations will have to be done with the admin user.

Reply:
On Prod or Debug how will backend then read, write, or delete from the database while the code is live?

Reply:
Further, given that on debug/development builds the admin user will not be locked, the default admin credentials are usable by us to invoke these scripts. In production builds, however, the API credentials will be virtually unknown by anyone after the create_api_user script, thus any initial setup of records in the db (whether it be random, pre-set, or restored), will have to be done solely by the admin user.

Reply:
Why couldnt this be done with the API user. As of now when the API user is created they do not even have permissions to list users of the db nor view any other db besides smart-ledger. We shouldn't need root access to insert data that API user would end up doing regardless with API calls

Reply:
if the api credentials are not stored via environment variables how else would the backend establish a valid connection with the db?

Reply:
reverted the usage print statement back since that actually wasn't used as of yet until we figure out exactly how to use secrets (perhaps Docker secrets)